<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSoundPage</class>
 <widget class="QWidget" name="CSoundPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sound name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="soundNameLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="browsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="playPushButton">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="spawnCheckBox">
       <property name="text">
        <string>Spawn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="muteCheckBox">
       <property name="text">
        <string>Mute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="keepPitchCheckBox">
     <property name="text">
      <string>Keep original pitch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Emission percent:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NLQT::CEditRangeFloatWidget" name="emissionWidget" native="true"/>
   </item>
   <item>
    <widget class="NLQT::CAttribFloatWidget" name="gainWidget">
     <property name="title">
      <string>Sound volume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NLQT::CAttribFloatWidget" name="pitchWidget">
     <property name="title">
      <string>Sound pitch</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CEditRangeFloatWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CAttribFloatWidget</class>
   <extends>QGroupBox</extends>
   <header>attrib_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
